lib/sidekiq/instrument/worker.rb

```diff
@@ -62,6 +60,14 @@ def dd_tags(queue)
       ["queue:#{queue.name}"]
     end
 
+    def emit_stats(statter, method, stat)
```

Comment:
@mokpro don't we need `stat: nil` in the method signature? Otherwise users of emit status will have to pass `nil` for stats explicitly.